libportability-gfx/src/conv.rs

```diff
@@ -16,6 +16,7 @@ pub fn limits_from_hal(limits: Limits) -> VkPhysicalDeviceLimits {
         maxFramebufferHeight: limits.max_texture_size as _, //TODO
         maxTexelBufferElements: limits.max_texture_size as _, //TODO
         maxTessellationPatchSize: limits.max_patch_size as _,
+        maxPushConstantsSize: 0x100, //TODO
```

Comment:
That's quite alot. The required minimum is 128 (0x80) which we should also expose here. Even some Vulkan drivers don't support more.